src/tokens.scss

```diff
@@ -0,0 +1,4 @@
+:host {
+  --ds-auro-table-background-color: var(--ds-color-container-secondary-default, $ds-color-container-secondary-default);
+  --ds-auro-table-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
```

Comment:
These are missing the `#{...}` around the SCSS variable in order to make them compile correctly.

Reply:
I don't think we need to do this in the `token.scss` file. It hasn't been done or pointed out in any of the other repos you or I have worked on.